packages/ember-runtime/lib/mixins/container_proxy.js

```diff
@@ -55,7 +55,14 @@ export default Mixin.create({
    @param {Object} options
    @return {any}
    */
-  lookup: containerAlias('lookup'),
+  lookup() {
+    let container = get(this, '__container__');
+    let result = container.lookup(...arguments);
+    if (result && result.owner === undefined) {
+      result.owner = this;
```

Comment:
at the very least some underscores, at the very best non-enumerable.

But why can't we just use __container__ ? it seems like its the same thing...